dash_docs/chapters/dash_vtk/other/index.py

```diff
@@ -0,0 +1,176 @@
+import dash_html_components as html
+import dash_vtk
+from dash_docs import tools
+from dash_docs import styles
+from dash_docs import reusable_components as rc
+
+examples = tools.load_examples(__file__)
+
+layout = html.Div([
+    rc.Markdown(
+    '''
+    # Other Dash VTK Components
+
+    ## Mesh
+
+    This element is a helper on top of __PolyData__ which has a Python helper function that goes with it which will help you map a __vtkDataSet__ into a single property of the __Mesh__ element.
+
+    ```py
+    def Mesh(**kwargs):
+        return dash_vtk.PolyData(
+            **kwargs.get('state').get('mesh'),
+            children=[
+                dash_vtk.[kwargs.get('state').get('field').get('location')]([
+                    dash_vtk.DataArray(
+                      **kwargs.get('state').get('field'),
+                    )
+                ])
+            ]
+        )
+    ```
+
```

Comment:
```suggestion

Note that we assume that `state.field` exists when we use `kwargs.get('state').get('field').get('location')` in the snippet above. However, `state.field` is optional, so when it's not available the `children` is not created, and `kwargs.get('state').get('field').get('location')` would not be needed.

```

@jourdain is that what you meant?

Reply:
perfect!